lib/cli/interactive-setup/dashboard-login.js

```diff
@@ -26,15 +26,15 @@ const steps = {
 
 module.exports = {
   async isApplicable(context) {
-    const { isConsole, configuration, options, serviceDir } = context;
+    const { isDashboard, configuration, options, serviceDir } = context;
 
-    if (isConsole) {
-      context.inapplicabilityReasonCode = 'CONSOLE_CONTEXT';
+    if (!serviceDir) {
+      context.inapplicabilityReasonCode = 'NOT_IN_SERVICE_DIRECTORY';
       return false;
     }
 
-    if (!serviceDir) {
-      context.inapplicabilityReasonCode = 'NOT_IN_SERVICE_DIRECTORY';
+    if (!isDashboard) {
```

Comment:
Let's stick to the original resolution order (first confirm that dashboard is applicable and then whether we're in service etc.)

Additional note: I see that in `org` check we have this _other_ order. I think it's fine to fix it over there (it's definitely more logical to first check for general applicability and then follow with more specific steps)

Reply:
> I see that in org check we have this other order.

I am not sure what you meant by this but I may have addressed this in my most recent push 😅